source/conf.py

```diff
@@ -290,7 +290,7 @@
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
 man_pages = [
-    ('index', 'perconamonitoringandmanagement', u'Percona Monitoring and Management Documentation',
+    ('reference/pmm-admin', 'pmm-admin', u'Percona Monitoring and Management Documentation',
```

Comment:
This creates a man page that can be added to the PMM Client distribution package somehow.